pkg/utils/template/template_test.go

```diff
@@ -0,0 +1,89 @@
+package template
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
```

Comment:
Blank line should be removed?

Reply:
Ah that's because I use **goimports** for local development. [Here's](https://goinbigdata.com/goimports-vs-gofmt/) how it differs from **gofmt**